src/stock_str.rs

```diff
@@ -275,6 +275,42 @@ pub enum StockMessage {
 
     #[strum(props(fallback = "Download maximum available until %1$s"))]
     DownloadAvailability = 100,
+
+    #[strum(props(fallback = "Inbox"))]
+    Inbox = 103,
```

Comment:
I would keep "Incoming messages" here, otherwise connectivity view looks like this:

### Inbox

- :green_circle: **Inbox**: Connected
- ...

i.e. "Inbox" appears on two levels, which will lead to confusion.

Reply:
i just wanted to have the same wording for the "Account settings" which has the headlines "Inbox" and "Outbox".

however, maybe it is better to change these headlines in the ui to "Incoming messages" and "Outgoing messages" as well, not sure. but independingly of that, i agree, for the connectivit view, the old headlines fit better.